Source/GUI/MainWindow.cpp

```diff
@@ -69,6 +69,10 @@ void MainWindow::initialiseWidgets()
   connect(m_btnRerollPrediciton, &QPushButton::clicked, this, &MainWindow::rerollPredictor);
   m_btnRerollPrediciton->setEnabled(false);
 
+  m_btnAutoReroll = new QPushButton(tr("Auto Reroll\n(rerolls until it finds a valid starter(s))"));
```

Comment:
`Reroll until the next desired starter(s)`

Bikeshedding, I know, but I just don't like the it pronoun here.  Also, technically, they are all valid, it`s just they are unwanted,